src/ObjWriting/ObjContainer/IWD/IWDWriter.cpp

```diff
@@ -0,0 +1,106 @@
+#include "IWDWriter.h"
+
+#include "../minizip/zip.h"
+
+#include <algorithm>
+#include <iostream>
+
+class IWDWriterImpl final : public IWDWriter
+{
+public:
+    explicit IWDWriterImpl(const std::string& iwdFileName, ISearchPath* assetSearchPath)
+        : m_filename(iwdFileName),
+          m_asset_search_path(assetSearchPath),
+          m_files(0)

```

Comment:
Changed to use the clear() method.

Reply:
You should not need to explicitly initialize a vector. If you just omit any initialization it will call the default constructor of `std::vector` which does properly initialize it as empty vector